lawrenceChege/iReporter
# 162188916 edit a redflag

# #162188916 modify a red flag
### What does this PR do?
```
This PR creates the tests for the API endpoints to modify a red flag
the test for getting all should pass
```
#### Description of Task to be completed?
```
Test if the app is editing a redflag
```
#### How should this be manually tested?
``` clone the repository 
create a virtual environment 
activate the virtual environment
install requirements using pip install -r requirements.txt
run pytest n the test for editing a red flag should pass
or run the app via python run.py and test on postman
```
### What are the relevant pivotal tracker stories?
[162188916](https://www.pivotaltracker.com/story/show/162188916)
#### Screenshots (if appropriate)
### Questions:
